src/cargo/util/context/mod.rs

```diff
@@ -2030,18 +2030,22 @@ impl ConfigError {
         }
     }
 
+    fn is_missing_field(&self) -> bool {
+        self.error.to_string().contains("missing field")
```

Comment:
Not a blocker. Just wonder if it is possible to compare with error types instead of strings.

Reply:
Have Added a custom `MissingField` errot to match.